recipes/starshot_editoria11y/recipe.yml

```
@@ -0,0 +1,10 @@
+name: Editoria11y support
+type: Starshot
+description: Installs and configures editoria11y module.
```

This should be a little more generic.

```suggestion
description: Installs and configures tools to help authors make their content accessible.
```